container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -3536,3 +2150,1641 @@ configuration:
+  options:
+    description: Expiration for generated certificates (in days)
+    default: 10950
+- name: CF_USB_BROKER_SERVER_CERT
+  options:
+    secret: true
+    ca: INTERNAL_CA_CERT
+    role_name: cf-usb-cf-usb
+    description: PEM-encoded broker server certificate.
+  type: certificate
+- name: CF_USB_BROKER_SERVER_CERT_KEY
+  options:
+    secret: true
+    description: PEM-encoded broker server key.
+- name: CF_USB_PASSWORD
+  options:
+    secret: true
+    example: password
+    description: The password for access to the Universal Service Broker.
+  type: password
+- name: CLUSTER_ADMIN_AUTHORITIES
+  options:
+    default: scim.write,scim.read,openid,cloud_controller.admin,clients.read,clients.write,doppler.firehose,routing.router_groups.read,routing.router_groups.write
+    description: The Oauth2 authorities available to the cluster administrator.
+    required: true
+- name: CLUSTER_ADMIN_PASSWORD
+  options:
+    secret: true
+    description: The password for the cluster administrator.
+    required: true
+- name: CLUSTER_BUILD
+  options:
+    default: 2.0.2
+    description: "'build' attribute in the /v2/info endpoint"
+    required: true
+- name: CLUSTER_DESCRIPTION
+  options:
+    default: SUSE Cloud Foundry
+    description: "'description' attribute in the /v2/info endpoint"
+    required: true
+- name: CLUSTER_NAME
+  options:
+    default: SCF
+    description: "'name' attribute in the /v2/info endpoint"
+    required: true
+- name: CLUSTER_VERSION
+  options:
+    default: 2
+    description: "'version' attribute in the /v2/info endpoint"
+    required: true
+- name: CONSUL_CLIENT_CERT
+  options:
+    secret: true
+    ca: INTERNAL_CA_CERT
+    description: PEM-encoded consul client certificate
+    required: true
+  type: certificate
+- name: CONSUL_CLIENT_CERT_KEY
+  options:
+    secret: true
+    description: PEM-encoded consul client key
+    required: true
+- name: CONSUL_SERVER_CERT
+  options:
+    secret: true
+    ca: INTERNAL_CA_CERT
+    alternative_names:
+    - "*.consul-consul-agent-set"
+    - server.dc1.{{.KUBERNETES_NAMESPACE}}.svc.{{.KUBERNETES_CLUSTER_DOMAIN}}
+    description: PEM-encoded server certificate
+    required: true
+  type: certificate
+- name: CONSUL_SERVER_CERT_KEY
+  options:
+    secret: true
+    description: PEM-encoded server key
+    required: true
+- name: CONSUL_SERVER_ENCRYPT_KEY
+  options:
+    secret: true
+    immutable: true
+    description: Consul server encryption key
+    required: true
+  type: password
+- name: CREDHUB_ENCRYPT_KEY
+  options:
+    secret: true
+    immutable: true
+    description: Credhub encryption key
+    required: true
+  type: password
+- name: CREDHUB_LOG_LEVEL
+  options:
+    description: >
+      The log level for credhub. This parameter is not supposed to be set
+      by the user. To change the cluster's log level set the parameter
+      `LOG_LEVEL` instead, from which this one is derived.
+  type: environment
```

Comment:
This is still correct, right? I'm kinda lost on what `type` means at this point; it seems to be the generator type?